src/openstudio_app/CMakeLists.txt

```diff
@@ -202,6 +209,21 @@ elseif( UNIX )
     VERBATIM
   )
 
+  add_custom_command(TARGET ${target_name}
+    POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${icudtl} $<TARGET_FILE_DIR:${target_name}>/
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${qweb_resources} $<TARGET_FILE_DIR:${target_name}>/
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${qweb_resources_100} $<TARGET_FILE_DIR:${target_name}>/
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${qweb_resources_200} $<TARGET_FILE_DIR:${target_name}>/ )
```

Comment:
Locally you need these `icudtl.dat` and the `resources/qtwengine*.pak`. These were already INSTALLED which explained why it (used) to work when installed but not when running OS App locally.